CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
```

Comment:
Do we really use 3.10 features? Is it widely available - eg on odroids and such?

Reply:
Version 3.10 is required to have the variables `OpenGL_GLX_FOUND` and `OpenGL_EGL_FOUND`. What is the operating system used by odroids? If it is Ubuntu 18.04 or later then it will be fine (https://packages.ubuntu.com/bionic/cmake).

Reply:
Ubuntu 18.04 is too recent - most of these board comes with distros from 2016 or so. We can keep them on the make path though, and only use CMake for newer distros

Reply:
You can update CMake from the official site if needed but I understand your point. No reason to drop the Makefile right now and I am happy with the idea of using CMake for newer distros.